legacy/include/distconv/tensor/halo_packing_cuda.hpp

```diff
@@ -149,7 +149,7 @@ struct PackAndPutBlockFunctor {
   static constexpr HaloTraversalOpGroup group = HaloTraversalOpGroup::BLOCK;
   static constexpr bool has_pre_grid = false;
   static constexpr bool has_post_grid = false;
-  static constexpr bool modifies_tensor = false;
+  static constexpr bool modifies_tensor = true;
```

Comment:
This should be `false`, but it causes compile errors at https://github.com/LLNL/DiHydrogen/blob/f37b2c031a5857643030c74fac1197c37e71814a/legacy/include/distconv/tensor/halo_cuda.hpp#L658 Everything looks correct, but maybe NVCC does something incorrect with `std::add_const`?